buglist.cgi

```diff
@@ -659,7 +659,7 @@ if ($order) {
     "Importance" => ["priority", "bug_severity"],
     "Assignee"   => ["assigned_to", "bug_status", "priority", "bug_id"],
     "Last Updated" =>
-      ["changeddate", "bug_status", "priority", "assigned_to", "bug_id"],
+      ["changeddate DESC", "bug_status", "priority", "assigned_to", "bug_id"],
```

Comment:
Basically the list data is comma joined into the ORDER BY section of the SQL. So it would become 

```ORDER BY changeddate DESC, bug_status, priority, assigned_to, bug_id```

So it first sorts by changeddate and the other columns are sorted if similar timestamps are next to each other in the list. But come to think about it, you dont really want secondary columns when sorting by a timestamp since you would always want the results to be in strict chronological order. And you would not have two rows with the same exact timestamp normally anyway. 

I reduce the order by for last updated and also add a comment as to why we don't want/need the additional columns.